objects/src/transaction/proven_tx.rs

```diff
@@ -352,6 +351,7 @@ impl TxAccountUpdate {
 
 impl Serializable for TxAccountUpdate {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        self.transaction_id.write_into(target);
```

Comment:
Do we actually need to serialize the transaction ID? It can be re-computed from the info in the `ProvenTransaction`.